packages/terra-date-picker/tests/jest/DatePicker.test.jsx

```diff
@@ -130,3 +130,23 @@ it('throws error on missing locale prop in Base', () => {
     expect(e.message).toContain('add locale prop to Base component');
   }
 });
+
+it('should render a date picker with onCalendarButtonClick', () => {
+  const handleOnCalendarButtonClick = () => {};
+  const datePicker = shallow(
+    <IntlProvider locale={locale} messages={messages}>
+      <DatePicker name="date-input" onCalendarButtonClick={handleOnCalendarButtonClick} utcOffset={0} />
+    </IntlProvider>,
+  );
+  expect(datePicker).toMatchSnapshot();
+});
+
+it('should render a date picker with onInputFocus', () => {
+  const handleOnInputFocus = () => {};
+  const datePicker = shallow(
+    <IntlProvider locale={locale} messages={messages}>
```

Comment:
Is there a reason we aren't using the TerraI18nProvider here?

Reply:
This is following the same way as the other components.